docs/redirectless.md

```diff
@@ -0,0 +1,98 @@
+# Redirectless support
```

Comment:
In the main README we refer to this as just "the widget". Can we update the references here to match the wording?